lib/text/ttml_text_parser.js

```diff
@@ -403,19 +417,37 @@ shaka.text.TtmlTextParser.parseCueRegion_ = function(regionElement, styles) {
 
   let results = null;
   let percentage = null;
+  let globalWidth = null;
+  let globalHeight = null;
+  if (globalExtent) {
+    percentage = TtmlTextParser.percentValues_.exec(globalExtent);
+    results = percentage || TtmlTextParser.pixelValues_.exec(globalExtent);
```

Comment:
Please avoid re-using results and percentage. In this case it looks like having a "global" and "local" version would make it easier to understand when each form of the value is being used.